src/hermes/commands/deposit/invenio.py

```diff
@@ -43,13 +49,81 @@ def map_metadata(ctx: CodeMetaContext):
     ctx.update(metadata_path, deposition_metadata)
 
 
-def _request_json(url: str) -> dict:
-    """Request an URL and return the JSON response as dict."""
+def deposit(click_ctx: click.Context, ctx: CodeMetaContext):
+    """Make a deposition on an Invenio-based platform.
+
+    This function can:
+
+    - Update the metadata of an existing record
+    - Update the metadata and files of an existing record y creating a new version
```

Comment:
```suggestion
    - Update the metadata and files of an existing record by creating a new version
```

**concern:** But looking into the code, it says it can't.

Reply:
Is cdee4a8bc362e42aa8746b9bd215f156b8dbcdd2 better?